jenkins.groovy

```diff
@@ -0,0 +1,65 @@
+pipeline
+{
+   agent { label 'linux' }
+
+   options
+   {
+      buildDiscarder logRotator(artifactDaysToKeepStr: '30', artifactNumToKeepStr: '100', daysToKeepStr: '30', numToKeepStr: '100')
+      timestamps()
+   }
+
+   tools
+   {
+     go 'Go'
```

Comment:
Is there are way to specify the Go version that will be used to build the project? That would be nice. :smile: 

The version can be found in the `go.mod` file: https://github.com/Graylog2/graylog-project-cli/blob/23c6bfc7e00e4439dc245680ff9277ff0f76c0bd/go.mod#L3

Reply:
I have to specify each version separately in the Jenkins admin config. Once that's set up, the user can select whatever Go versions I've made available. 

Right now, it's using v1.14.3. It still builds it the same, doesn't it? It's backwards compatible, surely?

What versions would be good to make available?

Reply:
Okay, cool. That's great. I think it makes sense to make 1.14.x available. I am switching to 1.14 anyway for the next release. Thanks!